starter-cli/src/test/groovy/io/micronaut/starter/cli/feature/acme/CliBaseSpec.groovy

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2017-2020 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.micronaut.starter.cli.feature.acme
+
+import io.micronaut.context.BeanContext
+import io.micronaut.core.io.socket.SocketUtils
+import io.micronaut.starter.cli.CommandFixture
+import io.micronaut.starter.cli.CommandSpec
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.testcontainers.Testcontainers
+import org.testcontainers.containers.GenericContainer
+import org.testcontainers.containers.wait.strategy.HostPortWaitStrategy
+import org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy
+import org.testcontainers.containers.wait.strategy.WaitAllStrategy
+import org.testcontainers.utility.MountableFile
+import spock.lang.AutoCleanup
+import spock.lang.Ignore
+import spock.lang.Shared
+
+import java.time.Duration
+
+@Ignore
```

Comment:
This should probably be an abstract class so `@Ignore` is not required

Reply:
Cleaned up in 0839f10